src/components/Heroes/index.jsx

```diff
@@ -12,7 +12,7 @@ import {
   sum,
   abbreviateNumber,
 } from '../../utility';
-import columns from './columns';
+import { RankColumns } from './RankColumns.ts';
```

Comment:
it's kind of confusing to use function that has name in `PascalCase` as plain function, they're usually used as components in react apps. Does it make sense either to rename this fn to `camelCase` or use it as component in this file?

Reply:
Yep really good point. camelCasing much better idea. Appreciate all the feedback btw

Reply:
Btw, KCD has recently published [a post](https://kentcdodds.com/blog/dont-call-a-react-function-component) about the problems that can occur with this

Reply:
Sorry I don't quite understand what you mean. This has no internal state that it manages, so the above article doesn't seem appropriate.

If, however, you're talking about the relationship between expecting it to be a component when PascalCasing it, as opposed to it being a raw function when camelCased, then yes I absolutely agree.

Reply:
> If, however, you're talking about the relationship between expecting it to be a component when PascalCasing it, as opposed to it being a raw function when camelCased, then yes I absolutely agree.

that's the main point, yes (and changing the name to match the convention was and still the only thing I have suggested 😄)

> This has no internal state that it manages, so the above article doesn't seem appropriate.

it does not have internal state yet, but it can be later added by any other developer that could be unaware about this component being used as a plain fn somewhere in the code

Reply:
Agreed. Thanks for the suggestions, have implemented them all